ethereum/datastructures/src/main/java/tech/pegasys/teku/datastructures/util/AttestationUtil.java

```diff
@@ -150,13 +147,17 @@ public static AttestationProcessingResult is_valid_indexed_attestation(
     SSZList<UnsignedLong> indices = indexed_attestation.getAttesting_indices();
 
     List<UnsignedLong> bit_0_indices_sorted =
-        indices.stream().sorted().distinct().collect(Collectors.toList());
+        indices.stream().sorted().distinct().collect(toList());
```

Comment:
Oh, is it allowed by our codestyle? I mean methods static import.
(would be just happy is yes 👍)

Reply:
Yes it is as long as the method name isn't too generic (so this is fine but a static import of `valueOf` is not).  I it passes error prone and aids readability go for it. :)